backend/api/management/commands/generate_parquets.py

```diff
@@ -52,7 +52,7 @@ def handle(self, *args, **kwargs):
             pa.field("cell_saver_ml", pa.uint32(), nullable=False),
             pa.field("overall_units", pa.uint16(), nullable=False),        # computed/stored
 
-            pa.field("los", pa.decimal128(6, 2), nullable=True),
+            pa.field("los", pa.uint32(), nullable=True),
```

Comment:
This is defined as decimal from `clinical_los` and from the intelvia.py models file. Are you sure this change is right and necessary?